nodepool/nl01.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -85,6 +85,9 @@ providers:
       - name: vyos-1.1.8-20190407
         config-drive: true
         connection-type: network_cli
+      - name: esxi-6.7.0-20190802001-STANDARD
+        config-drive: true
+        username: root
```

Comment:
We can keep this for now, to confirm we can boot and log into the node but we really do want to remove this root user. The reason is, for all other images we use the zuul user, so I'd like to keep this image the same as that.

Also, we usually use the root user as the login for infra-root users, which allows us to access the system outside of the zuul user.